parquet_test.go

```diff
@@ -1026,3 +1026,158 @@ var quickCheckConfig = quick.Config{
 func quickCheck(f interface{}) error {
 	return quickCheckConfig.Check(f)
 }
+
+func TestParquetAnyValueConversions(t *testing.T) {
+	// This test runs conversions to/from any values with edge case schemas.
+
+	type obj = map[string]any
+	type arr = []any
```

Comment:
I apologise in advance for these local aliases, definitely optimising here for writing at the cost of reading so happy to remove these if asked.

Reply:
This doesn't bother me, thanks for calling it out 👍